quay.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	quay "github.com/sebrandon1/go-quay/lib"
+)
+
+const (
+	bearerToken = "wdSLOoT8K3pa4BQhsTnWxLSobTWMNF4X67p4VmU1"
```

Comment:
This value can't be checked/hardcoded in, needs to be passed in via config or something.

Reply:
thanks! I agree!!! I created config folder and there I put bearerToken, clientID, namespace and repository.